code/obj/machinery/cell_charger.dm

```diff
@@ -92,3 +92,9 @@
 	. = ..()
 	if(Obj == src.charging)
 		src.charging = null
+
+/obj/machinery/cell_charger/get_desc(dist)
+	. = ..()
+	if(dist > 2 || !charging)
```

Comment:
Probably not necessary? I know it'd confuse the hell out of me if the description of something changed based on distance.

Reply:
Sure. One moment